_posts/2024-05-28-edb-and-kasten.markdown

```diff
@@ -0,0 +1,295 @@
+cluster-example-2            0/1     Running           0          4s
+cluster-example-2            1/1     Running           0          5s
+cluster-example-2            1/1     Running           0          5s
+cluster-example-3-join-vm6d9   0/1     Pending           0          0s
+cluster-example-3-join-vm6d9   0/1     Pending           0          5s
+cluster-example-3-join-vm6d9   0/1     Init:0/1          0          5s
+cluster-example-3-join-vm6d9   0/1     PodInitializing   0          10s
+cluster-example-3-join-vm6d9   1/1     Running           0          11s
+cluster-example-3-join-vm6d9   0/1     Completed         0          13s
+cluster-example-3-join-vm6d9   0/1     Completed         0          15s
+cluster-example-3-join-vm6d9   0/1     Completed         0          16s
+cluster-example-3              0/1     Pending           0          0s
+cluster-example-3              0/1     Pending           0          0s
+cluster-example-3              0/1     Init:0/1          0          0s
+cluster-example-3              0/1     PodInitializing   0          4s
+cluster-example-3              0/1     Running           0          5s
+cluster-example-3              0/1     Running           0          5s
+cluster-example-3              1/1     Running           0          6s
+cluster-example-4-join-rtpxf   0/1     Pending           0          0s
+cluster-example-4-join-rtpxf   0/1     Pending           0          5s
+cluster-example-4-join-rtpxf   0/1     Init:0/1          0          5s
+cluster-example-4-join-rtpxf   0/1     PodInitializing   0          9s
+cluster-example-4-join-rtpxf   0/1     Completed         0          10s
+cluster-example-4-join-rtpxf   0/1     Completed         0          12s
+cluster-example-4-join-rtpxf   0/1     Completed         0          13s
+cluster-example-4              0/1     Pending           0          0s
+cluster-example-4              0/1     Pending           0          0s
+cluster-example-4              0/1     Init:0/1          0          0s
+cluster-example-4              0/1     PodInitializing   0          6s
+cluster-example-4              0/1     Running           0          7s
+cluster-example-4              0/1     Running           0          7s
+cluster-example-4              1/1     Running           0          8s
+cluster-example-4-join-rtpxf   0/1     Terminating       0          21s
+cluster-example-3-join-vm6d9   0/1     Terminating       0          43s
+cluster-example-4-join-rtpxf   0/1     Terminating       0          21s
+cluster-example-3-join-vm6d9   0/1     Terminating       0          43s
+cluster-example-3              1/1     Running           0          28s
+cluster-example-2              1/1     Running           0          50s
+cluster-example-4              1/1     Running           0          9s
+```
+
+### Check your data are back.
+
+As you restore everything you also restore the client, connect to the client and check you have your data.
```

Comment:
As Kasten restores everything that was backed up by default it has also restored the client. Connect to the client and check the data.